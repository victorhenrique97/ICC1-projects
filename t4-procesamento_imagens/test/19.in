2
img1.pgm
conv3.mat

