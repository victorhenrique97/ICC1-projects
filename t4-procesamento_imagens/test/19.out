P2
24 7
15
9 8 8 8 8 9 10 9 9 8 8 10 11 9 9 8 8 11 12 9 9 9 9 11 
8 7 7 7 7 9 9 5 5 5 6 10 9 3 3 3 4 10 10 1 2 2 1 9 
9 7 8 8 9 10 9 5 7 8 10 12 10 3 6 8 10 14 11 1 6 5 0 9 
9 6 7 8 9 10 9 5 6 8 11 13 10 2 4 8 12 15 10 0 3 3 1 9 
8 7 8 9 9 10 9 5 7 8 10 12 10 3 6 8 10 14 10 0 7 11 10 11 
8 7 9 9 9 10 9 5 5 5 6 10 9 3 3 3 4 10 9 2 10 13 11 10 
9 8 9 9 9 9 9 9 9 8 8 10 11 9 9 8 8 10 10 8 10 10 9 9 
